kedro/framework/cli/project.py

```diff
@@ -53,8 +53,8 @@
 PIPELINE_ARG_HELP = """Name of the registered pipeline to run.
 If not set, the '__default__' pipeline is run."""
 PARAMS_ARG_HELP = """Specify extra parameters that you want to pass
-to the context initializer. Items must be separated by comma, keys - by colon,
-example: param1:value1,param2:value2. Each parameter is split by the first comma,
+to the context initializer. Items must be separated by comma, keys - by colon or equal sign,
```

Comment:
An incredibly minor note here - we want to use UK English for this one: https://kedro.readthedocs.io/en/stable/contribution/documentation_contributor_guidelines.html
```suggestion
to the context initialiser. Items must be separated by comma, keys - by colon or equals sign,
```